api/auth/index.ts

```diff
@@ -1,13 +1,15 @@
 import { AUTH, xuiNode } from '@hmcts/rpx-xui-node-lib'
 import * as express from 'express'
 import { NextFunction, Request, Response } from 'express'
+import { EnhancedRequest } from '../lib/models'
 import {getConfigValue} from '../configuration'
 import {COOKIE_ROLES} from '../configuration/references'
 import { http } from '../lib/http'
 
+
 //TODO: once both todo's below are complete, we can remove this file
 
-const successCallback = (req: Request, res: Response, next: NextFunction) => {
+const successCallback = (req: EnhancedRequest, res: Response, next: NextFunction) => {
```

Comment:
What is this change for, @udaydenduluri33? It looks entirely unrelated to skip link functionality... 🤔

Reply:
@udaydenduluri33 can you respond to @paul-graham's query as I am looking to get this PR approved asap. Many thanks.

Reply:
Enhanced request gives capability to use the parameters with that type. 
Express request/session is used to store the passport values for OIDC. These are already done on MC. Not sure how AO missed it.